clientconn.go

```diff
@@ -1143,8 +1144,13 @@ func (cc *ClientConn) Close() error {
 	<-cc.balancerWrapper.serializer.Done()
 
 	for ac := range conns {
-		ac.tearDown(ErrClientConnClosing)
+		wg.Add(1)
+		go func(adrCon *addrConn) {
```

Comment:
I'd just shadow `ac` here instead of making up some new shorthand.

Reply:
Done